app/src/main/java/com/duckduckgo/app/pixels/remoteconfig/AndroidBrowserConfigFeature.kt

```diff
@@ -181,9 +179,20 @@ interface AndroidBrowserConfigFeature {
     @Toggle.DefaultValue(DefaultFeatureValue.FALSE)
     fun vpnMenuItem(): Toggle
 
-    @Toggle.DefaultValue(TRUE)
+    /**
+     * @return `true` when the remote config has the global "useUnifiedOmnibarLayout" androidBrowserConfig
+     * sub-feature flag enabled
+     * If the remote feature is not present defaults to `true`
+     */
+    @Toggle.DefaultValue(DefaultFeatureValue.TRUE)
     fun useUnifiedOmnibarLayout(): Toggle
 
-    @Toggle.DefaultValue(FALSE)
+    /**
+     * @return `true` when the remote config has the global "splitOmnibar" androidBrowserConfig
+     * sub-feature flag enabled
+     * If the remote feature is not present defaults to `false`
+     */
+    @Toggle.DefaultValue(DefaultFeatureValue.FALSE)
+    @Toggle.InternalAlwaysEnabled
```

Comment:
```suggestion
    @Toggle.DefaultValue(DefaultFeatureValue.INTERNAL)
```
Not forcing makes it easier to test.

Reply:
That said, I would also suggest keeping it hidden/disabled by default until it's closer to completion (the duplicate buttons now might make it confusing for someone internally stumbling upon it).

Reply:
@LukasPaczos They'd need to explicitly enable it in the settings. I wouldn't be concerned about it. But keeping it off for now is fine.